cliconfig/config.go

```diff
@@ -15,19 +15,19 @@ import (
 )
 
 const (
-	// Where we store the config file
-	CONFIGFILE     = "config.json"
-	OLD_CONFIGFILE = ".dockercfg"
+	// CONFIGFILE is the name of config file
+	CONFIGFILE    = "config.json"
```

Comment:
Of course, there is also a struct in this unit called `ConfigFile`. How about `ConfigFilename`?